vector/src/fdroid/java/im/vector/app/nightly/NightlyProxy.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.nightly
+
+import javax.inject.Inject
+
+class NightlyProxy @Inject constructor() {
+    fun updateIfNewReleaseAvailable() {
```

Comment:
could be `fun updateIfNewReleaseAvailable()  = Unit`

Reply:
https://github.com/vector-im/element-android/pull/6629/commits/4dd5fa7051b373d1a622c6158723fd370d0f8749